### PR TITLE
upgrade aws-ebs-csi-controller to 1.0.0

### PR DESCRIPTION
**Pull request by ettiee**
Replaces #212

### CODE REVIEW

File: modules/cluster/addons/kustomize/overlays/aws-ebs-csi-driver/kustomization.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
-- github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/alpha/?ref=v0.9.0
+- github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-0.10
```
Comment (errm) [inline modules/cluster/addons/kustomize/overlays/aws-ebs-csi-driver/kustomization.yaml:4]:
```suggestion
- github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.0
```

??

Comment (ettiee) [reply modules/cluster/addons/kustomize/overlays/aws-ebs-csi-driver/kustomization.yaml]: Ah thanks - spuriously placed decimal point! 8cb8814 ..

**Comment by ettiee** [comment]
38a04a9 addresses test failure 
Screenshot 2021-06-03 at 10 01 58

but we should fix the kubebench checks properly  in #186

**Review by errm — APPROVED** [review]
LGTM

**Comment by errm** [comment]
Because of the way that we apply the configs here `kubectl apply` it looks like this change will leave behind a number of service accounts that are removed or renamed, perhaps we need to mention them in the upgrade notes.

**Comment by ettiee** [comment]
> Because of the way that we apply the configs here with `kubectl apply` it looks like this change will leave behind a number of service accounts that are removed or renamed, perhaps we need to mention them in the upgrade notes.

Ah thanks good point - I'll add to the notes on the release

**Pull request merged**